cryptodoc/src/08_signatures.rst

```diff
@@ -314,7 +314,7 @@ The signature generation algorithm works as follows:
 
    **Input:**
 
-   -  ``m``: raw bytes to sign (EMSA1 encoded data)
+   -  ``m``: raw bytes to sign (the hash-code ``H``, must be already truncated if needed)
```

Comment:
> What is the hash-code H here?

This is the same as in ISO-14888-3.

> EMSA1 was mentioned before, why has this been removed.?

Because it was wrong before (or at least inaccurate).

> Moreover, is there any reference to EMSA1 and what it means?

EMSA1 means basically just sign the hash. Many of the signing algorithms like ECKCDSA only support this instead of any padding.
With Botan 3 this algorithms expect now only the hash functions instead of mentioning EMSA1. See also https://github.com/randombit/botan/blob/master/doc/deprecated.rst#deprecated-functionality.

Reply:
> > What is the hash-code H here?
> 
> This is the same as in ISO-14888-3.
> 
> > EMSA1 was mentioned before, why has this been removed.?
> 
> Because it was wrong before (or at least inaccurate).

It is, however, mentioned for the other signature algorithms. Moreover, it is mentioned below. 

> 
> > Moreover, is there any reference to EMSA1 and what it means?
> 
> EMSA1 means basically just sign the hash. Many of the signing algorithms like ECKCDSA only support this instead of any padding. With Botan 3 this algorithms expect now only the hash functions instead of mentioning EMSA1. See also https://github.com/randombit/botan/blob/master/doc/deprecated.rst#deprecated-functionality.

There should be a reference and/or a definition if this term is used.

Reply:
Added an explanation what `H` is.

Reply:
Added also a reference to EMSA1.